src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -329,28 +331,32 @@ private BlockPos getFloor(BlockPos position)
     private AIState clearStep()
     {
         WorkOrderBuild wo = job.getWorkOrder();
-        if(wo.isCleared())
+        if (wo.isCleared())
         {
             return AIState.BUILDER_STRUCTURE_STEP;
         }
 
         BlockPos coordinates = job.getSchematic().getBlockPosition();
-        Block worldBlock = world.getBlockState(coordinates).getBlock();
+        Block    worldBlock  = world.getBlockState(coordinates).getBlock();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'worldBlock' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)